chal6.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title></title>
+  </head>
+  <body>
+    <input id = "input"><button onclick ="start()">Check</button>
+    <script type="text/javascript">
+      function start() {
+        var input =  document.getElementById("input").value;
+        var values = input.split("")
+        var ifIntOrFloat = Number.isInteger(values.length/2);
+        for (var i = 0; i < values.length; i++) {
+          var valueIsInt = Number.isInteger(i/2);
+          if(ifIntOrFloat === true){
```

Comment:
You can just do `if (ifIntOrFloat)` since `ifIntOrFloat === true` is the same as `ifIntOrFloat`. A better variable name would be `isNumeric`.